<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>836</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Axis Selection</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="RadLabel" >
          <property name="text" >
           <string>Radius Variable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="radiusVariable" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="DistLabel" >
          <property name="text" >
           <string>Distance Variable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="distanceVariable" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="AngleLabel" >
          <property name="text" >
           <string>Angle Variable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="angleVariable" />
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="planeDistanceVariable" />
      </item>
      <item>
       <widget class="QLineEdit" name="bTransXVariable" />
      </item>
      <item>
       <widget class="QLineEdit" name="bTransYVariable" />
      </item>
      <item>
       <widget class="QLineEdit" name="bTransZVariable" />
      </item>
      <item>
       <widget class="QLineEdit" name="bScaleXVariable" />
      </item>
      <item>
       <widget class="QLineEdit" name="bScaleYVariable" />
      </item>
      <item>
       <widget class="QLineEdit" name="bScaleZVariable" />
      </item>
      <item>
       <widget class="QLineEdit" name="clipType" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5" >
     <property name="title" >
      <string>Comparison View</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="alignCircles" >
        <property name="text" >
         <string>Enable Comparison View</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="CompLabel" >
          <property name="text" >
           <string>Comparison Variable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="thVariable" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Reverse</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="reverseRadius" >
        <property name="text" >
         <string>Radius</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Reference Circle Properties</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="ThickLabel" >
          <property name="text" >
           <string>Thickness</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="CircleThickness" >
          <property name="decimals" >
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="RadScaleLabel" >
          <property name="text" >
           <string>Radius Scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="referenceCirclesSize" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="distScaleLabel" >
          <property name="text" >
           <string>Distance Scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="distanceBetween" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="NumGridLabel" >
          <property name="text" >
           <string>Number Of Grids</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numberOfGrids" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="NumCirLabel" >
          <property name="text" >
           <string>Number Of Circles</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="numberOfCircles" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Axis Properties</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="AxThicknessLabel" >
          <property name="text" >
           <string>Thickness</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="AxisThickness" />
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>61</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLineEdit" name="ApplyPressed" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="sliceTypeGroup" >
     <property name="title" >
      <string>Clip</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="enableClip" >
        <property name="text" >
         <string>Enable Clip</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="1" >
         <widget class="QLabel" name="ClipTypeLabel" >
          <property name="text" >
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QComboBox" name="sliceType" />
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QStackedWidget" name="sliceFrame" >
          <widget class="QWidget" name="page" />
          <widget class="QWidget" name="page_2" />
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
